portal/src/components/UU/UU.tsx

```diff
@@ -0,0 +1,278 @@
+                            <motion.li
+                                key={node.id}
+                                initial={{ y: prefersReducedMotion ? 0 : -40, opacity: 0 }}
+                                animate={{
+                                    y: prefersReducedMotion ? 0 : 0,
+                                    opacity: 1,
+                                    transition: { duration: 0.3 },
+                                }}
+                                layout
+                                transition={{
+                                    type: "tween",
+                                    duration: 0.3,
+                                }}
+                                exit={{ y: prefersReducedMotion ? 0 : 40, opacity: 0, transition: { duration: 0.2 } }}
+                            >
+                                <GatsbyLink to={`#${node.slug}`} className="jkl-link">
+                                    {node.frontmatter.title}
+                                </GatsbyLink>
+                            </motion.li>
+                        ))}
+                </ul>
+
+                {filteredNodes.map((node) => (
+                    <article key={node.id} className="uu-article">
+                        <header className="uu-article__header">
+                            <h3 className="uu-article__header--heading" id={node.slug}>
+                                {node.frontmatter.title}
+                            </h3>
+                            <ul className="uu-article__header--tags" aria-label="Artikkel tags">
+                                {node.frontmatter.tags.map((t) => (
+                                    <li key={t} className="uu-article__header__tag">
+                                        {tagMap[t]}
+                                    </li>
+                                ))}
+                            </ul>
+                        </header>
+                        <FormatProvider>
+                            <MDXRenderer>{node.body}</MDXRenderer>
+                        </FormatProvider>
+                        {(node.frontmatter.wcagRules?.length || node.frontmatter.links?.length) && (
+                            <>
+                                <h4>Lenker</h4>
+                                <ul className="uu-article__links uu-article__links--wcag">
+                                    {node.frontmatter.wcagRules?.map((rule) => {
+                                        const criteria = getCriteriaById(rule);
+                                        return (
+                                            <li key={rule}>
+                                                <Link
+                                                    title={`${criteria.title} på w3.org`}
+                                                    href={`https://www.w3.org/TR/WCAG21/#${criteria.w3OrgId}`}
+                                                    target="_blank"
```

Comment:
Ikke at det er så viktig for oss nå, men bruker vi blank må du vel ha med noreferer også

Reply:
Enig i at det er noe vi bør kjøre som standard.